doc/kubernetes/modules/ROOT/pages/storage/embedded-infinispan.adoc

```diff
@@ -1,26 +0,0 @@
-= Configuring Embedded Infinispan
```

Comment:
@ahus1 We would still have the embedded-infinispan as part of the Keycloak store in its current shape, do we want to move this information to a different place instead ?